### DISCUSSION TITLE
SNMP inkomplett ? (tcp-mib/udp-mib)

### DISCUSSION THREAD

**Discussion by maverick3005** [Freetz-NG/freetz-ng#670] [Allgemein]
Ich sehe bei snmp im compile-log dieses hier:

```
configure: WARNING: mib module error
configure: WARNING: mib module 'mibII' requires module 'tcp-mib' but someone told me to compile without it
configure: WARNING: mib module error
configure: WARNING: mib module 'mibII' requires module 'udp-mib' but someone told me to compile without it
```

Soll das so sein ? Wäre es nicht sinnvoller den kompletten Modul-Support hierzu mit einzubauen ?

**Comment by fda77** [comment]
Das Flash in der Fritzbox ist begrenzt, daher macht man nur Ding an die man auch wirklich braucht. Ob du sie brauchst musst du wissen, für mich sind MIBs men in black
